oak_abi/build.rs

```diff
@@ -14,10 +14,10 @@
 // limitations under the License.
 //
 
+// Keep clippy from complaining about a needless call to `Default::default()`.
+#[allow(clippy::needless_update)]
```

Comment:
I actually think it's better to remove the default call; if we add extra fields to `ProtoOptions`, it would be good if the compiler tells us we need to update this file too. Anyways, up to you.

Reply:
I would actually argue the exact opposite 😅 . When you add new options here, this is most likely due to a new feature that you need to support for a particular use case, it should not require changes to existing code (if possible). In other words, putting the call to `Default::default()` here allows for some degree of backwards compatibility.

Reply:
Yes I understand, and certainly if this was a third party crate I would agree. I just think that in our case it's nice for the compiler to tell us all the places that require update, just to double check we are not missing anything. your choice anyways :)

Reply:
Aah, I was under the impression that third parties that want to integrate with Oak would also use `oak_utils` to generate code? Or is it just that we're not designing for that explicitly yet?